.github/workflows/ci.yml

```diff
@@ -0,0 +1,33 @@
+name: Check
+
+on:
+  push:
+    branches:
+      - master
+      - releases/v[0-9]+.[0-9]+.[0-9]+
+  pull_request:
+    types: [opened, synchronize]
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+jobs:
+  validate_copyright:
+    name: Copyright
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+      with:
+        show-progress: false
+    - name: Install dependencies
+      run: python3 -m pip install --user python-debian
+    - name: Validate copyright file with python-debian
+      run: python3 ./utils/check_copyright.py
+    - name: Validate copyright file with debian-control-linter
+      uses: tibetiroka/debian-control-linter-action@v1
+      with:
+        type: 'debian/copyright'
+        file: 'copyright'
+        preset: 'strict'
+        enable: 'trailingSpace,upstreamContactStyle,urlForceHttps,fieldNameCapitalization,maintainerNameFullStop,urlExists,copyrightSourceStyle'
```

Comment:
Are there any checks you don't want to run? If not, then these should be the default so that if you add a check to your action you don't have to go here and add it manually.

Reply:
This is the same config as ES. Most of the stuff is enabled via the `strict` preset; the checks in `enable` are rules we generally adhere to, but isn't mandated by the specs. )